pkg/sink/sink.go

```diff
@@ -173,6 +173,12 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 func (r Sink) merge(et []triggersv1.EventListenerTrigger, trItems []*triggersv1.Trigger) ([]*triggersv1.Trigger, error) {
 	triggers := trItems
 	for _, t := range et {
+		// if ELT serviceAccountName not set, take that as assuming the user if fine with using the default SA;
+		// but no longer allow the EventListener SA and its permissions to influence what can be created
+		serviceAccountName := "default"
```

Comment:
Curious - alternatively, instead of explicitly setting `default` SA, do you think it would also work to conditionally set `TriggerSpec.serviceaccount` based on presence of `t.ServiceAccountName` after 201, so we could leave it blank when omitted? Would that would achieve the same result, since the SA would default to `default`?

Reply:
not exactly @iancoffey .... the intent on always setting it explicitly here is so that the event listener sink's use of k8s impersonation kicks in, and as a result, any subsequent creation of objects are from the start done using permissions *OTHER* than the permissions of the event listener sink.